charts/prometheus-postgres-exporter/templates/role.yaml

```
@@ -8,7 +8,7 @@ metadata:
     chart: {{ template "prometheus-postgres-exporter.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
-{{- if .Values.rbac.pspEnabled }}
+{{- if (and .Values.rbac.pspEnabled (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy")) }}
```

Extra `(` `)` should not be necessary.
```suggestion
{{- if and .Values.rbac.pspEnabled (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
```

---

Corrected, thank you.